src/Chapter17/Listing17.09.SlicingWithSpan.cs

```diff
@@ -0,0 +1,41 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter17.Listing17_09;
+
+using System.Diagnostics;
+
+public class Program
+{
+    public static void Main()
+    {
+        #region INCLUDE
+        string[] languages = new [] {
+            "C#", "COBOL", "Java",
+            "C++", "TypeScript", "Python",};
+
+        // Create a Span<string> from the arrays first 3 elements.
+        Span<string> languageSpan = languages.AsSpan(0,2);
+        languages[0] = "R";
+        Trace.Assert(languages[0] == languageSpan[0]);
```

Comment:
Do we really want to use Trace.Assert? It can pop a message box on failure blocking execution. VS does some magic to rewrite this method to avoid the hang but I don't think this is what we want. Could we do something like a Console.WriteLine to demonstrate the condition?

Reply:
Added a custom Assert() method.